code/drasil-code/lib/Language/Drasil/Code/Imperative/Descriptions.hs

```diff
@@ -74,9 +74,9 @@ inputConstructorDesc = do
       icDesc True = "checking " ++ pAndS ++ " on the input"
       dl = defList g
   return $ "Initializes input object by " ++ stringList [ 
-    ifDesc ("get_input" `elem` dl),
-    idDesc ("derived_values" `elem` dl),
-    icDesc ("input_constraints" `elem` dl)]
+    ifDesc (genICFuncName GetInput `elem` dl),
```

Comment:
So this code is a tiny improvement over what was there before. Why is `dl` still a `String` ? Does it have to be?